### ISSUE TITLE
Traning datasets about Mapfree  and DL3DV

### ISSUE THREAD

**Issue by zhangshuoneu** [Livioni/OmniVGGT-official#13]
Thanks for the great work on 3D geometry foundation models! The comprehensive experiments and analysis are very impressive.

Regarding the data, I noticed that your training set significantly overlaps with the CUT3R dataset. Did you refer to CUT3R for the preprocessing of the training data, as well as for the Mapfree and DL3DV datasets?

Also, given that these datasets use COLMAP for dense depth estimation and contain a lot of noise, how did you address this? Did you clean the noisy data, or did you generate your own dense depth labels?

**Comment by Livioni** [comment]
We followed the processing script provided by Cut3r. For details, please refer to [dl3dv.py](https://github.com/Livioni/OmniVGGT-official/blob/b49bd02454824f25a7d2f65ebd2a03fdaca8a26d/omnivggt/datasets/dl3dv.py) and [mapfree.py](https://github.com/Livioni/OmniVGGT-official/blob/b49bd02454824f25a7d2f65ebd2a03fdaca8a26d/omnivggt/datasets/mapfree.py).
